bitmovin/resources/models/encodings/drms/playready_drm.py

```diff
@@ -1,16 +1,36 @@
 from .drm import DRM
+from bitmovin.resources.enums import PlayReadyMethod
+from bitmovin.utils import Serializable
 
 
-class PlayReadyDRM(DRM):
+class PlayReadyDRM(DRM, Serializable):
 
     def __init__(self, key_seed, kid, method, la_url=None, outputs=None, id_=None, custom_data=None,
                  name=None, description=None):
         super().__init__(id_=id_, custom_data=custom_data, outputs=outputs, name=name, description=description)
+        self._method = None
+
         self.method = method
         self.keySeed = key_seed
         self.kid = kid
         self.laUrl = la_url
 
+    @property
+    def method(self):
+        return self._method
+
+    @method.setter
+    def method(self, value):
+        if value is None:
```

Comment:
Because this is a required field, I this is not necessary. So if it is None the InvalidTypeError will be thrown.